src/query/sql/src/planner/binder/project.rs

```diff
@@ -117,6 +117,32 @@ impl Binder {
                         scalar,
                         index: item.index,
                     })
+                } else if let ScalarExpr::WindowFunction(win) = &item.scalar {
+                    if let Some(column) = bind_context
+                        .windows
+                        .window_functions_map
+                        .get(&win.agg_func.display_name)
+                    {
+                        let window_info = &bind_context.windows.window_functions[*column];
+                        let column_binding = ColumnBinding {
+                            database_name: None,
+                            table_name: None,
+                            column_name: win.display_name(),
+                            index: window_info.aggregate_function.index,
+                            data_type: Box::new(window_info.aggregate_function.scalar.data_type()?),
+                            visibility: Visibility::Visible,
+                        };
+                        let scalar = BoundColumnRef {
+                            span: None,
+                            column: column_binding,
+                        }
+                        .into();
+                        return Ok(ScalarItem {
+                            scalar,
+                            index: item.index,
+                        });
+                    }
+                    Err(ErrorCode::Internal("Invalid window function"))
```

Comment:
Should conduct a DFS like `GroupingChecker`, because Window Function could be nested in other exprs.

Like:

```select
select rank() over (xxx) + 1 from t;
```